setup.py

```diff
@@ -29,8 +32,20 @@ def get_version(*file_paths):
     author_email="contact@edunext.co",
     url="https://github.com/eduNEXT/eox-core",
     description="eduNEXT Openedx extensions",
-    long_description="",
-    install_requires=[],
+    long_description=README,
+    long_description_content_type='text/x-rst',
+    classifiers=[
```

Comment:
I think we should include the edx-platform versions supported so far. @morenol

Reply:
I think that we only can add classifiers from this list

Reply:
Got it.